source/getting-started/routes-and-templates.md

```diff
@@ -0,0 +1,166 @@
+
+```app/templates/about.hbs
+<h2>About Super Rentals</h2>
+
+<p>The Super Rentals website is a delightful project created to explore Ember.<br>
+  By building a property rental site, we can simultaneously imagine traveling<br>
+  AND building Ember applications simultaneously.</p>
+```
+
+## A Contact Page
+
+Let's create another static page with details for contacting the company.  Once again, we'll start by generating a route, a route handler, and a template.
+
+```shell
+ember g route about
+```
+
+We see that our generator has created a `contact` route and corresponding route handler in `app/routes/contact.js`.  We will be using the `contact` template, so the `contact` route does not need any additional changes.
+
+In `contact.hbs`, we can add the details for contacting our Super Rentals HQ:
+
+```app/templates/contact.hbs
+<p>Super Rentals Representatives would love to help you choose a destination or answer
+any questions you may have.</p>
+
+<p>Contact us today:</p>
+
+<p>
+  Super Rentals HQ<br>
+  1212 Test Address Avenue<br>
+  Testington, OR 97233
+</p>
+
+<p>(503)555-1212</p>
+
+<p>superrentalsrep@superrentals.com</p>
+```
+
+Now we have completed our second static page.  If we enter the URL,  `http://super-rentals/contact`, we arrive on our contact page.
+
+## Navigating with links and the {{link-to}} helper
+
+We really don't want users to have to know our URLs in order to move around our site, so let's add some navigational links at the bottom of each page.  Let's make a contact link on the about page and an about link on the contact page.
+
+Ember has built-in **helpers** that provide functionality such as linking to other routes.  Here is how to use the {{link-to}} helper in our code:
+
+```app/templates/about.hbs
+<h2>About Super Rentals</h2>
+
+<p>The Super Rentals website is a delightful project created to explore Ember.<br>
+  By building a property rental site, we can simultaneously imagine traveling<br>
+  AND building Ember applications simultaneously.</p>
+
+{{#link-to 'contact'}}Click here to contact us.{{/link-to}}
+```
+
+The helper takes an argument with the name of the route to link to, in this case, `contact`.  When we look at our about page, we now have a link to our contact page.
+
+![super rentals about page screenshot](../../images/routes-and-templates/ember-super-rentals-about.png)
+
+Now, we'll add one to link to our about page so we can navigate from back and forth from `about` to `contact`.
+
+```app/templates/contact.hbs
+<p>Super Rentals Representatives would love to help you <br>
+  choose a destination or answer any questions you may have.</p>
+
+<p>Contact us today:</p>
+
+<p>
+  Super Rentals HQ<br>
+  1212 Test Address Avenue<br>
+  Testington, OR 97233
+</p>
+
+<p>(503)555-1212</p>
+
+<p>superrentalsrep@superrentals.com</p>
+
+{{#link-to 'about'}}About{{/link-to}}
+```
+
+## An Index Page
```

Comment:
Done.  Also updated other "Page" titles to "Route" (About and Contact).


On Tue, Sep 15, 2015 at 12:39 PM, Michael Kaiser-Nyman <
notifications@github.com> wrote:

> In source/getting-started/routes-and-templates.md
> <https://github.com/emberjs/guides/pull/738#discussion_r39555038>:
>
> > +<p>Contact us today:</p>
> > +
> > +<p>
> > +  Super Rentals HQ<br>
> > +  1212 Test Address Avenue<br>
> > +  Testington, OR 97233
> > +</p>
> > +
> > +<p>(503)555-1212</p>
> > +
> > +<p>superrentalsrep@superrentals.com</p>
> > +
> > +{{#link-to 'about'}}About{{/link-to}}
> > +```
> > +
> > +## An Index Page
>
> How about "An Index Route"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emberjs/guides/pull/738/files#r39555038>.
>